python/mxnet/initializer.py

```diff
@@ -656,7 +656,7 @@ def _init_weight(self, _, arr):
         c = (2 * f - 1 - f % 2) / (2. * f)
         for i in range(np.prod(shape)):
             x = i % shape[3]
-            y = (i / shape[3]) % shape[2]
+            y = (i // shape[3]) % shape[2]
```

Comment:
there is a `_init_bilinear` method in the base Initializer class - https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/initializer.py#L212 . Is this class redundant? 

Also I see that many of the initializers do not have tests. Could you please add a test for this change by creating a new file `test_initializer.py` in the tests folder. Tests for other initializers can be added to the same file later.